autotst/species.py

```diff
@@ -1062,28 +1062,16 @@ def set_chirality(self, chiral_center_index, stero="R"):
         return self
 
     def calculate_symmetry_number(self):
-        from rmgpy.qm.symmetry import PointGroupCalculator
-        from rmgpy.qm.qmdata import QMData
-
-        atom_numbers = self.ase_molecule.get_atomic_numbers()
-        coordinates = self.ase_molecule.get_positions()
-
-        qmdata = QMData(
-            groundStateDegeneracy=1,  # Only needed to check if valid QMData
-            numberOfAtoms=len(atom_numbers),
-            atomicNumbers=atom_numbers,
-            atomCoords=(coordinates, str('angstrom')),
-            energy=(0.0, str('kcal/mol'))  # Only needed to avoid error
-        )
-        settings = type(str(''), (), dict(symmetryPath=str(
-            'symmetry'), scratchDirectory="."))()  # Creates anonymous class
-        pgc = PointGroupCalculator(settings, self.smiles, qmdata)
-        pg = pgc.calculate()
-        #os.remove("{}.symm".format(self.smiles))
-
-        if pg is not None:
-            symmetry_number = pg.symmetry_number
-        else:
-            symmetry_number = 1
 
-        return symmetry_number
+        numbers = self.ase_molecule.numbers
+        positions = self.ase_molecule.positions
+
+        mol = RMGMolecule()
```

Comment:
small clarification, is `mol` different from `self.rmg_molecule`? can we just use `self.rmg_molecule` instead of making a new object that might need more memory!

Reply:
If ‘self.rmg_molecule’ has coords, I think we should use that. If not, maybe use the ‘update_coords_from’ method to create rmg mol from ase mol?